students/kaczmarek_katarzyna/lesson_05_lists/comma_code.py

```diff
@@ -0,0 +1,17 @@
+def stringify(items):
+    if len(items) > 0:
```

Comment:
tutaj lepiej użwać `if items` co odpowiada `if len(items)>0 and items is not None`